vertical-pod-autoscaler/pkg/recommender/input/history/history_provider_test.go

```diff
@@ -51,54 +51,106 @@ func getDefaultPrometheusHistoryProviderConfigForTest() PrometheusHistoryProvide
 	}
 }
 
-func (m *mockPrometheusClient) GetTimeseries(query string) ([]Timeseries, error) {
-	args := m.Called(query)
-	var returnArg []Timeseries
+type mockPrometheusAPI struct {
```

Comment:
Out of curiosity, is there no mock/fake API provided by the prometheus libs?

Reply:
Not that I'm aware of